ci/use_wheels_from_prs.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) 2025, NVIDIA CORPORATION.
+
+RAPIDS_PY_CUDA_SUFFIX=$(rapids-wheel-ctk-name-gen "${RAPIDS_CUDA_VERSION}")
+
+# download wheels, store the directories holding them in variables
+LIBRMM_WHEELHOUSE=$(
+  RAPIDS_PY_WHEEL_NAME="librmm_${RAPIDS_PY_CUDA_SUFFIX}" rapids-get-pr-wheel-artifact-github librmm 1896 cpp
+)
```

Comment:
Oops. I changed too many rmm -> librmm.
```suggestion
LIBRMM_WHEELHOUSE=$(
  RAPIDS_PY_WHEEL_NAME="librmm_${RAPIDS_PY_CUDA_SUFFIX}" rapids-get-pr-wheel-artifact-github rmm 1896 cpp
)
```